net/socketman/Makefile

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2016 Cucumber WiFi
+#
+# This is free software, licensed under the GNU General Public License v2.
+
+export CT_ARCH="-D__OPENWRT__"
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=SocketMan
+PKG_SOURCE_PROTO:=git
+PKG_MAINTAINER:=Andrew Margarit <build_bot@cucumberwifi.io>
+PKG_LICENSE:=GPL-2
+PKG_SOURCE_VERSION:=02dabd47aa5297db19c9e1d1ef716d2e2a4b772b
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)
+PKG_SOURCE_URL:=https://github.com/cucumber-tony/SocketMan.git
+PKG_SOURCE:=$(PKG_NAME).tar.gz
+PKG_VERSION:=v1
+
```

Comment:
* PKG_SOURCE should contain a version identifier (and prefer .xz over .gz)
* PKG_LICENSE should be either GPL-2.0 or GPL-2.0+ (standard names).
* Add PKG_LICENSE_FILES:=LICENSE
* PKG_SUBDIR should also have a version ID (if possible)
* Need PKG_MD5SUM and PKG_HASH